### PR TITLE
enhancement/apply_pom_style

### PR DESCRIPTION
**Pull request by losete**
Pull Request made for applying the [Maven Code Style And Code Conventions](https://maven.apache.org/developers/conventions/code.html) on pom.xml files

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,118 +1,114 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (pablogalegoc) [inline pom.xml]: Following the [generic maven code style and conventions](https://maven.apache.org/developers/conventions/code.html)
Line wrapping: Always use a 120-column line width.
Same issue in the other pom.xml

File: domain/pom.xml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+  <artifactId>domain</artifactId>
+
+  <name>domain</name>
+  <description>domain</description>
+
+  <dependencies>
+    <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
```
Comment (pablogalegoc) [inline domain/pom.xml]: Over indented lines

File: application/pom.xml [hunk 3]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (pablogalegoc) [inline application/pom.xml]: Over indented file

**Review by pablogalegoc — CHANGES_REQUESTED** [review]

**Review by pablogalegoc — DISMISSED** [review]

**Comment by franrua** [comment]
`Applied requested changes` is not a good commit message, because is related with the PR. If I see the git log isolated, can't know what changes was asked for.

**Review by franrua — CHANGES_REQUESTED** [review]
Each pom section must has their header. You can see your partners pom files.

File: bootloader/pom.xml [hunk 4]
```diff
@@ -2,78 +2,85 @@
...
+      <artifactId>spring-boot-autoconfigure</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot</artifactId>
+    </dependency>
+  </dependencies>
+
+  <!-- Build Settings -->
+
```
Comment (franrua) [inline bootloader/pom.xml:31]: Part of the header is missing.  There are ending delimiters. Review reference pom.

```  
<!-- ///////////////// -->

  <!-- Build Settings -->
```

File: application/pom.xml [hunk 5]
```diff
@@ -2,32 +2,38 @@
...
+    <artifactId>hexagonal-architecture-poc</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>application</artifactId>
+
+  <name>application</name>
+  <description>application</description>
+
+  <!-- Dependencies -->
```
Comment (franrua) [inline application/pom.xml]: This section doesn't appear in the reference pom. Dependencies are included in Pom relationship section.

**Review by franrua — CHANGES_REQUESTED** [review]
Apply headers and delimiters using reference pom as guide

**Review by franrua — DISMISSED** [review]

**Review by pablogalegoc — DISMISSED** [review]

**Review by pablogalegoc — APPROVED** [review]

**Pull request merged**